src/help/changelog.md

```diff
@@ -8,7 +8,7 @@ In this release of the Broker, we made the following updates:
 
 
 #### Schema - Released DAIMS v1.2 {section=daims12release}
-Treasury released the final DATA Act Information Model Schema (DAIMS) v1.2. DAIMS v1.2 is a minor update of the schema and addresses some agency feedback, implements policy requirements, promote additional data standardization and reduce agency burden. The release is targeted for implementation in production for the submission of FY 2018 Quarter 3 data. Find out more information in the [DABS Resources](#/dabsResources) section.
+Treasury released the final DATA Act Information Model Schema (DAIMS) v1.2. DAIMS v1.2 is a minor update of the schema and addresses some agency feedback, implements policy requirements, promote additional data standardization and reduce agency burden. The release is targeted for implementation in production for the submission of FY 2018 Quarter 3 data. Find out more information in the [DABS Resources](#/Resources) section.
```

Comment:
Should this be lowercase? I thought our link was lowercase (I know case doesn't matter but we should be consistent)

Reply:
lowercased